janusgraph-codepipelines-ci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM maven:3-jdk-8
+ENV DEBIAN_FRONTEND noninteractive
+COPY settings-docker.xml /usr/share/maven/ref/
+ENV COMMIT 9165ee8407b3607f39031cdaeff1d15ba0fd4e9e
+ENV SRC_DIR /tmp/janusgraph-${COMMIT}
+RUN wget -q -O /tmp/jg.zip https://github.com/JanusGraph/janusgraph/archive/${COMMIT}.zip \
+    && unzip -q /tmp/jg.zip -d /tmp \
+    && mvn install \
+         --settings /usr/share/maven/ref/settings-docker.xml \
+         --threads 2.0C \
+         --batch-mode \
+         --file ${SRC_DIR}/pom.xml \
+         -DskipTests=true -Dmaven.javadoc.skip=true -Dmaven.compiler.showWarnings=false \
+    && rm -rf ${SRC_DIR}
```

Comment:
on second thought, it seems like I am not using this Dockerfile, so I will remove it.